### PR TITLE
Update feature list

### PR DESCRIPTION
**Pull request by fmauch**
Update the feature list with the new features from this branch

### CODE REVIEW

**Review by stefanscherzinger — APPROVED** [review]

File: ur_robot_driver/doc/features.md [hunk 1]
```diff
@@ -1,35 +1,31 @@
...
+| Cartesian twist-based control                         | yes                   |
+| Trajectory forwarding for execution on robot          | yes                   |
+| reporting of tcp wrench                               | yes                   |
+| reporting of tcp wrench in tcp link                   | yes                   |
+| pausing of programs                                   | yes                   |
+| continue trajectories after EM-Stop resume            | yes                   |
+| continue trajectories after protective stop           | yes                   |
+| panel interaction in between possible                 | yes                   |
+| get and set IO states                                 | yes               |
+| use tool communication on e-series                    | yes                   |
```
Comment (urrsk) [inline ur_robot_driver/doc/features.md:18]: Suggest a adding a link for this as well

File: ur_robot_driver/doc/features.md [hunk 2]
```diff
@@ -1,35 +1,31 @@
...
+| use the driver without a teach pendant necessary      | yes               |
+| support of CB2 robots                                 | -                     |
+| trajectory extrapolation on robot on missing packages | yes                   |
+| use ROS as drop-in for TP-programs                    | yes   |
```
Comment (urrsk) [inline ur_robot_driver/doc/features.md:22]: Suggest a adding a link for this as well

File: ur_robot_driver/doc/features.md (hunk 1 above)
Comment (fmauch) [reply ur_robot_driver/doc/features.md:18]: See fb6c3ee

File: ur_robot_driver/doc/features.md (hunk 2 above)
Comment (fmauch) [reply ur_robot_driver/doc/features.md:22]: See fb6c3ee

**Pull request merged**